firstvoices/backend/serializers/media_serializers.py

```diff
@@ -105,7 +105,9 @@ class AudioSerializer(CreateSiteContentSerializerMixin, MediaSerializer):
         },  # for local dev, settings for browseable api
     )
     original = MediaFileUploadSerializer(
-        validators=[SupportedFileType(mimetypes=["audio/wav", "audio/mpeg"])],
+        validators=[
+            SupportedFileType(mimetypes=["audio/wav", "audio/x-wav", "audio/mpeg"])
```

Comment:
Oh good call. Could you please add the rest of the wav mimetypes as well? `audio/wave, audio/wav, audio/x-wav, audio/x-pn-wav` https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Containers#wave_wav